src/main/resources/static/cas2-domain-events-api.yml

```diff
@@ -185,10 +185,26 @@ components:
         description:
           type: string
           example: 'More information about the application has been requested from the POM (Prison Offender Manager).'
+        statusDetails:
+          type: array
+          items:
+            $ref: '#/components/schemas/StatusDetailItem'
       required:
         - name
         - label
         - description
+    StatusDetailItem:
```

Comment:
I wonder if a better name would just be `StatusDetail` or `Cas2StatusDetail` to following existing naming convention. I don't think `Item` is needed?

Reply:
Changed the name of this property from `StatusDetailItem` to `Cas2StatusDetail` in a more recent commit.